src/test/java/stepdefinitions/StepDefinitions.java

```diff
@@ -1,66 +1,103 @@
+    @And("^I navigate to the Mobile Chapter page$")
+    public void iNavigateToTheMobileChapterPage() {
         ValoriHomePage valoriHomePage = new ValoriHomePage(driver);
         MobileChapterPage mobileChapterPage = new MobileChapterPage(driver);
         valoriHomePage.clickBusinessDropdown();
         valoriHomePage.clickMobileInDropDOwn();
-        assertTrue("Expecting the full stack header to be visible", mobileChapterPage.isMobilePageVisibile());
+        assertTrue("Expecting the Mobile automation header to be visible", mobileChapterPage.isMobilePageVisibile());
     }
 
-    @Then( "^I can read articles about the Mobile Chapter$" )
-    public void iCanReadAboutMobileChapter(){
+    @Then("^I can read articles about the Mobile Chapter$")
+    public void iCanReadAboutMobileChapter() {
         MobileChapterPage mobileChapterPage = new MobileChapterPage(driver);
         assertTrue(mobileChapterPage.areArticlesVisible());
     }
 
-    @And( "^I contact Valori$" )
-    public void iContactValori()throws Throwable{
+    @And("^I contact Valori$")
+    public void iContactValori() throws Throwable {
         MobileChapterPage mobileChapterPage = new MobileChapterPage(driver);
         ContactUsPage contactUsPage = new ContactUsPage(driver);
         mobileChapterPage.clickContactUsButton();
+        driver.navigate().refresh();
 
-        //switch to new tab
+        //switch to new tab.
+        // Dit zorgt ervoor dat er een tab wordt geopend naar innovation.valori.nl/neem-contact-op
+        //Echter moet de link https://innovation.valori.nl/neem-contact-op zijn om de pagina te bereiken.
         ArrayList<String> tabs = new ArrayList<String>(driver.getWindowHandles());
         driver.switchTo().window(tabs.get(1));
+
+        //nakijken of de juiste pagina is bereikt.
         assertTrue(contactUsPage.isTitleVisible());
     }
 
-    @Then( "^the contact us form is available$" )
+    @Then("^the contact us form is available$")
     public void theContactUsFormIsAvailable() throws Throwable {
-        //TODO IMPLEMENT THIS
-        //Verify the contact form fields are available.
-        throw new PendingException();
+        ContactUsPage contactUsPage = new ContactUsPage(driver);
+        //nakijken of het contactformulier aanwezig is.
+        assertTrue(contactUsPage.isFormAvailable());
+
     }
-}
 
+    @Given("^fill the contact form with the following data$")
+        public void fillContactForm(DataTable exampleTable) throws Throwable {
+            UtilitiesAka aka = new UtilitiesAka(driver);
+        ContactUsPage contactUsPage = new ContactUsPage(driver);
+        //Nakijken of de contactformulier velden aanwezig zijn en deze ook gelijk invullen.
+        List<Map<String, String>> list = exampleTable.asMaps(String.class, String.class);
+        for (int i = 0; i < list.size(); i++) {
+            aka.ElementAssertAndFill("//input[contains(@id,'firstname-')]",(list.get(i).get("First name")));
```

Comment:
Alle data die je nu invult doet wordt verder niks mee gedaan. Je vult wat in en doet verder geen checks of de ingevulde data wordt geaccepteerd. Webdriver laat je gewoon text typen ookal laat de front end dat niet perse toe, daarom wil je hier altijd aparte assertions op schrijven om te valideren dat de foutmeldingen worden getoond.